ui/packages/consul-ui/app/components/app/index.scss

```diff
@@ -58,15 +61,37 @@
 %main-nav-vertical-hoisted.is-active > label > * {
   @extend %main-nav-horizontal-action-active;
 }
+%footer,
 %main-nav-sidebar,
 %main-notifications,
 main {
   @extend %transition-pushover;
 }
+%footer {
+  position: absolute;
+  z-index: 50;
+
+  color: rgb(var(--tone-gray-400));
+  font-size: var(--typo-size-800);
+
+  width: 250px;
+  padding-left: 25px;
+}
+.app footer {
+  top: calc(100vh - 42px);
+  top: calc(max(100vh, 460px) - 42px);
+}
+html.has-partitions.has-nspaces .app footer {
+  top: calc(100vh - 42px);
+  top: calc(max(100vh, 640px) - 42px);
+}
```

Comment:
Awkwardly the `<footer>` used to be pushed down by the `<main>` content. But now, _ideally_ it needs to be pushed down by the left hand `<nav>` menu (which is _inside_ the black background).

Unfortunately the `<nav>` is currently absolutely positioned and therefore can't push other things. So instead of reworking the entire thing here, I opted for using `max` which is now _mostly_ supported by our browser matrix, apart from FF 72, 73, 74 in which case _only_ if folks squeeze there browser window up enough the version will overlap the menu 🤷 

At some point I may rearrange things here so the menu isn't taken out of the natural flow of the document using absolute, but now is not the time 😸